docs/api.md

```diff
@@ -67,7 +67,17 @@ Use `prettier.resolveConfig.sync(filePath [, options])` if you'd like to use syn
 
 The promise will be rejected if there was an error parsing the configuration file.
 
-If `options.useCache` is `false`, all caching will be bypassed.
+If `filePath` is not provided, the path of the configuration file in the current directory will be returned.
```

Comment:
The configuration file does not _have_ to be in the current directory, right? It could also be in some parent directory?

Also, what happens if you _do_ provide a `filePath`?

Reply:
Thanks, I fixed.